lib/utils.js

```diff
@@ -181,6 +183,30 @@ const nudgeCallCounts = async(logger, sids, nudgeOperator, writers) => {
   return {callsSP: null, calls: null, callsApp: null};
 };
 
+const isPrivateVoipNetwork = async(uri) => {
+  if (process.env.PRIVATE_VOIP_NETWORK_CIDR) {
+    try {
+      const arr = /sips?:.*@(.*):.*$/.exec(uri);
+      if (arr) {
+        const input = arr[1];
+        let addresses;
+        if (input.match(/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/)) {
+          addresses = [input];
+        } else {
+          addresses = await dns.resolve4(input);
+        }
+        const matcher = new CIDRMatcher(process.env.PRIVATE_VOIP_NETWORK_CIDR.split(','));
```

Comment:
this CIDRMatcher object is always going to be created with the same network CIDR.  Can we just create it once when the file is included?